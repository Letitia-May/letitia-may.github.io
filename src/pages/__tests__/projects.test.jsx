import React from 'react'
import renderer from 'react-test-renderer'
import { StaticQuery } from 'gatsby'
import Projects from '../projects'

beforeEach(() => {
  StaticQuery.mockImplementation(({ render }) =>
    render({
      site: {
        siteMetadata: {
          title: `Chloe Kuypers`,
          description: `My portfolio, built with Gatsby`,
          author: `@letitia-may`,
        },
      },
    })
  )
})

describe('Projects', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<Projects />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
