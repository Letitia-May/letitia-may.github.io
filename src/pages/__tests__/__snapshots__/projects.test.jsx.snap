// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Projects renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="header"
  >
    <div
      className="header_content"
    >
      <div
        className="link"
      >
        <a
          href="/"
          style={
            Object {
              "color": "white",
              "textDecoration": "none",
            }
          }
        >
          <p
            className="main_title"
          >
            Chloe
          </p>
          <p
            className="subtitle"
          >
            Web dev
          </p>
        </a>
      </div>
    </div>
  </div>
  <div
    className="page_content"
  >
    <div
      className="tile tile_main"
      onMouseEnter={[Function]}
      onMouseLeave={[Function]}
    >
      <h3
        className="heading"
      >
        Capricorn Records
      </h3>
    </div>
  </div>
  <div
    className="footer"
  >
    <div
      className="footer_content"
    >
      <div
        className="icons"
      >
        <a
          className="container"
          href="https://github.com/Letitia-May"
          rel="noopener noreferrer"
          target="_blank"
        >
          <i
            className="fab fa-github"
          />
        </a>
        <a
          className="container"
          href="https://au.linkedin.com/in/ckuypers"
          rel="noopener noreferrer"
          target="_blank"
        >
          <i
            className="fab fa-linkedin-in"
          />
        </a>
        <a
          className="container"
          href="https://twitter.com/letitia_may"
          rel="noopener noreferrer"
          target="_blank"
        >
          <i
            className="fab fa-twitter"
          />
        </a>
      </div>
      <div
        className="links"
      >
        <p>
          Â© 
          2019
          , Built with
           
          <a
            href="https://www.gatsbyjs.org"
            rel="noopener noreferrer"
            target="_blank"
          >
            Gatsby
          </a>
        </p>
        <p>
          Source available on
           
          <a
            href="https://github.com/Letitia-May/portfolio"
            rel="noopener noreferrer"
            target="_blank"
          >
            Github
          </a>
        </p>
      </div>
    </div>
  </div>
</div>
`;
